components/src/Toast/Toast.js

```diff
@@ -0,0 +1,121 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import theme from "../theme";
+import { Box } from "../Box";
+import { Alert } from "../Alert";
+
+const ANIMATION_DURATION = 2000; // in ms
+const TOAST_Y_MAX = "0px";
+const TOAST_Y_MIN = "-32px";
+const ACTIVE_Z_INDEX = 2;
+const INACTIVE_Z_INDEX = ACTIVE_Z_INDEX - 1;
+
+const SLIDE_IN_STYLES = {
+  transform: `translateY(${TOAST_Y_MIN})`,
+  transition: "transform 0.15s ease-out",
+  zIndex: ACTIVE_Z_INDEX
+};
+
+const SLIDE_OUT_STYLES = {
+  transform: `translateY(${TOAST_Y_MAX})`,
+  transition: "transform 0.9s ease-in",
+  zIndex: INACTIVE_Z_INDEX
+};
+
+const FADE_IN_STYLES = {
+  opacity: 1,
+  transition: "opacity 0.25s linear"
+};
+
+const FADE_OUT_STYLES = {
+  transition: "opacity 1s linear"
+};
+
+const AnimatedAlert = styled(Alert)(({ visible }) => ({
+  boxShadow: theme.shadows.medium,
+  minWidth: "200px",
+  maxWidth: "600px",
```

Comment:
just using this max width to restrict the characters per line.. I'm not doing anything to enforce the number or lines/max text in the code, I opted for just adding that usage info in docs.
 It seems like that would be a confusing if the component cut off text that was inputted and would make it difficult to deal with longer translations

Reply:
This makes sense to me.